db/data_migrate/20181109141112_fix_submission_entries_total_values.rb

```diff
@@ -0,0 +1,27 @@
+require 'progress_bar'
+
+Framework.all.find_each do |framework|
+  puts framework.short_name
+
+  total_value_field = framework.definition::Invoice.total_value_field
+
+  entries = SubmissionEntry
+            .validated
+            .invoices
+            .where(
+              "submission_entries.total_value <> \
+               TRANSLATE(submission_entries.data->>'#{total_value_field}', '$£ ,', '')::decimal"
+            )
+            .joins(submission: :framework)
+            .merge(Framework.where(short_name: framework.short_name))
+
+  bar = ProgressBar.new(entries.count)
+
+  entries.find_each do |entry|
+    total_value = BigDecimal(entry.data[total_value_field].tr('$£, ', ''))
```

Comment:
@leeky do you feel confident with the data that this can remain as is too?

Reply:
@tahb Yeah, all the existing bad data matches these characters.